src/components/icon/test/icon.e2e.ts

```diff
@@ -4,10 +4,49 @@ import { test } from '@utils/test/playwright';
 test.describe('icon: basic', () => {
   test('should not have visual regressions', async ({ page }) => {
     await page.goto(`/`);
-    
+
     // Wait for all SVGs to be lazily loaded before taking screenshots
     await page.waitForLoadState('networkidle');
 
     expect(await page.screenshot({ fullPage: true })).toMatchSnapshot(`icon-diff.png`);
   });
-});
\ No newline at end of file
+
+  test('some icons should flip when rtl', async ({ page }) => {
+    await page.goto(`/`);
+
+    const autoflip = page.locator('.auto-flip-chevrons [name=chevron-forward] .icon-inner');
+    const unflip = page.locator('.un-flip-chevrons [name=chevron-forward] .icon-inner');
+    await expect(autoflip).not.toHaveCSS('transform', /matrix\(-1/);
+    await expect(unflip).not.toHaveCSS('transform', /matrix\(-1/);
+
+    await page.evaluate(() => {
+      document.dir = 'rtl';
+    });
```

Comment:
Is it possible to start the test with this pre-configured, or is that not what we are trying to test?

Reply:
No, both the assertions before and after this are important. This tests the original bug, which is that things were working fine in either `ltr` or `rtl`, but the icons weren't changing if the `dir` changed.

Reply:
Ahhh ok makes sense. Thanks!